kyuubi-server/src/main/scala/org/apache/kyuubi/server/metadata/api/Metadata.scala

```diff
@@ -78,6 +78,7 @@ case class Metadata(
     engineState: String = null,
     engineError: Option[String] = None,
     endTime: Long = 0L,
+    priority: Int = 10,
```

Comment:
There's a magic code here. Default priority is worth to be extracted to explicit static value , and further mentioned in config docs. Or it's confusing for user and developers reviewing the UTs.

Reply:
let's at least document it should be consistent with the metadata table DDL definition